app/core/models/ocelot/stanzas/CalculationStanza.scala

```diff
@@ -63,21 +65,43 @@ sealed trait Operation {
     labels.value(ref).getOrElse("")
   }
 
-  def unsupportedOperation(operationName: String)(arg1: Any, arg2: Any): Option[String] = {
+  def valueAsOption(arg: String, labels: Labels): Option[String] = labelReference(arg).fold[Option[String]](Some(arg)){ref =>
```

Comment:
think value and valueAsOption could be replaced by a single fn value returning an Option[String]

Reply:
Done